storage/internal/benchmarks/main.go

```diff
@@ -225,7 +236,12 @@ func parseFlags() {
 func main() {
 	log.SetOutput(os.Stderr)
 	parseFlags()
-	rand.Seed(time.Now().UnixNano())
+
+	go func() {
+		if err := http.ListenAndServe("localhost:8080", nil); err != nil {
```

Comment:
What does this do as a stand-alone? add a comment?

Reply:
Whoops, this was for obtaining pprof samples. I removed it; I'll open a PR separately to see if it makes more sense to wrap specific parts of the code with cpu profiling and take periodic memory samples, or else re-add this back in for ad-hoc profile collecting.